code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1651,6 +1651,9 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
 	..()
 
 /mob/living/carbon/human/proc/do_cpr(mob/living/carbon/human/H)
+	if(H == src)
+		to_chat(src, "<span class='warning'>You cannot perform CPR on yourself!</span>")

```

Comment:
WELL WHY THE HELL NOT?! THIS IS BS SHITAINER REEEEEEEEEEEEEEEE

Reply:
Agreed. Literally unplayable. Changed my mind. Do not merge ever.

Reply:
What do you mean I can't punch myself in the chest or run into a wall with enough force to compress my chest and kickstart my heart